### PR TITLE
PCM-3024 + PCM-3025

### PR DESCRIPTION
**Pull request by jtrantin**
@vrathee @gandhikeyur @engineersamuel  Please review.

https://projects.engineering.redhat.com/browse/PCM-3024
https://projects.engineering.redhat.com/browse/PCM-3025

Will add some screens to the JIRA.

### CODE REVIEW

File: app/cases/controllers/solrQueryInput.js [hunk 1]
```diff
@@ -97,19 +99,39 @@ export default class SolrQueryInputController {
...
 
             Promise.all(promises).then(function () {
+                $scope.setupAutocompletion();
                 $scope.$broadcast(CASE_EVENTS.focusSearchInput);
             });
         };
 
+        $scope.setupAutocompletion = () => {
+            if(RHAUtils.isNotEmpty(SOLRGrammarService.accounts)) {
+                SOLRGrammarService.accounts.forEach(account => {
```
Comment (engineersamuel) [inline app/cases/controllers/solrQueryInput.js]: This assumes accounts is never undefined.

Comment (engineersamuel) [inline app/cases/controllers/solrQueryInput.js]: I mean to say, lodash/each is typically safer, though there isn't necessarily an issue here, as long as accounts is guaranteed to be defined or [].

**Comment by engineersamuel** [comment]
Reviewed!

File: app/cases/controllers/solrQueryInput.js (hunk 1 above)
Comment (jtrantin) [inline app/cases/controllers/solrQueryInput.js]: hmm.. There is a check for emptiness just above the `.forEach`, so I think undefined or null don't break this.

Comment (engineersamuel) [inline app/cases/controllers/solrQueryInput.js]: I meant you could get rid of that isNot Empty and do `each(SOLRGrammarService.accounts, ...)`  but not a big deal, could save a line though.

Comment (jtrantin) [inline app/cases/controllers/solrQueryInput.js]: OK.. I see there are conflicts, so I will fix this while fixing merge conflicts.

**Comment by jtrantin** [comment]
Resolved conflicts + added Samuel's suggestion

**Comment by vrathee** [comment]
Reviewed!

**Comment by renujhamtani** [comment]
the screens look good!

**Pull request merged**